src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -1421,6 +1437,23 @@ def load_separator_trend_description(
     ]
 
 
+def load_controller_trend_description(
```

Comment:
Do you remember if this is really being used in production code or it is here just to appease the automatic test running gods?

Reply:
Hmm good point, I didn't check it. Maybe the other ones could be removed too.

Reply:
If it is used just for tests removing it will break the test.. the test helper must be updated to preferentially use the autoloaders.

Reply:
Maybe just updating the test parameters will help

Reply:
I updated #216.